syntax = "proto3";

import "hq/dto/Security.proto";
import "hq/dto/Ticker.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.notify";
option java_outer_classname = "TickerNotifyProto";
message TickerNotify {
    Security security = 1;
    Ticker ticker=2; // 逐笔列表
}